src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,231 @@
+  const renderEditorChoice = () => {
+    if (!editorChoice?.isEditorChoice) return null
+
+    return (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <Images.EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
+
+  const renderBlockchains = () => {
+    if (!blockchains || blockchains.length === 0) return null
+
+    const visibleBlockchains = blockchains.slice(0, MAX_VISIBLE_BLOCKCHAINS)
+
+    return (
+      <div className={styles.blockchains}>
+        <span className={styles.blockchainLabel}>
+          {i18n?.blockchainLabel || 'BLOCKCHAIN(S):'}
+        </span>
+        <div className={styles.blockchainIcons}>
+          {visibleBlockchains.map((blockchain, index) => (
+            <BlockchainIcon key={index} blockchain={blockchain} />
+          ))}
+          {showRemainingCount && (
+            <div className={styles.more}>
+              <span className={styles.moreCount}>+{remainingCount}</span>
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.imageContainer}>
+        {image ? (
+          <GameImage
+            src={image?.src}
+            alt={image?.alt}
+            fallback={image?.fallback}
+            className={styles.image}
+          />
+        ) : null}
+      </div>
+
+      <div className={styles.body}>
+        <div className={styles.content}>
+          <div className={styles.header}>
+            {renderEditorChoice()}
+            <span className={classNames(styles.title)}>{title}</span>
+          </div>
+
+          <div className={styles.info}>
+            <div className={styles.badges}>
+              {version ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  Version {version}
+                </Sticker>
+              ) : null}
+              {earlyAccess ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  Early Access
+                </Sticker>
+              ) : null}
+              <Sticker styleType="neutral" variant="outlined">
+                {title}
+              </Sticker>
+              {playerCount ? (
+                <Sticker styleType="neutral" variant="outlined">
+                  {playerCount}
+                </Sticker>
+              ) : null}
+            </div>
+
+            {renderBlockchains()}
+          </div>
+        </div>
+
+        <div className={styles.actions}>
+          <div className={styles.socialLinks}>
+            {socialLinks?.map((link, index) => (
+              <Button
+                key={index}
+                type="secondary"
+                size="icon"
+                onClick={() => window.open(link.url, '_blank')}
+              >
+                <link.IconButton className={styles.socialIcon} />
+              </Button>
```

Comment:
this should be a separate component and we should use a similar structure as on the `gamePage` in the store. It should take a Link component and have a default value as an `a` tag. You can see an example for the `Link` prop in `src/components/NavBar/index.tsx`

Reply:
I can put the link, but for a new component, I don't think it's necessary as I'm using the button component already,y which has a variant called "Icon Button".